pkg/odo/cli/version/version.go

```diff
@@ -31,8 +31,9 @@ var versionExample = ktemplates.Examples(`
 
 // VersionOptions encapsulates all options for odo version command
 type VersionOptions struct {
-	// clientFlag indicates if the user only wants client information
+	// Flags
```

Comment:
I find the information in the removed comment helpful. I would suggest to not remove it.

Reply:
Same as https://github.com/openshift/odo/pull/5251#discussion_r759990571